server/src/Controller/Channel/ChannelController.js

```diff
@@ -0,0 +1,99 @@
+import {
+  create,
+  deleteDoc,
+  findOne,
+  findMany,
+  update,
+} from '../../Repositories/genericRepository';
+import successHandler from '../../libs/routes/successHandler';
+import { channelModel } from '../../Model';
+
+
+const getChannel = async (req, res, next) => {
+  try {
+    const { id } = req.params;
+    const result = await findOne(channelModel, id);
+    if (!result) {
+      const err = new Error('Channel does not exists');
+      next(err);
+    }
+    res
+      .status(200)
+      .send(successHandler('Channel is here', result));
+  } catch (error) {
+    next(error);
+  }
+};
+
+const getAllChannels = async (req, res, next) => {
+  try {
+    const result = await findMany(channelModel);
```

Comment:
This API can have parameters like
1. user - user for whom all channels need to be provided ( this we can get from authentication middleware - for this PR it can be hardcoded for sample user maybe)
2. workspace - workspace for which all channels need to be provided